pyproject.toml

```diff
@@ -1189,6 +1189,7 @@ module = [
     "tests.sentry.snuba.test_query_subscription_consumer",
     "tests.sentry.snuba.test_tasks",
     "tests.sentry.spans.grouping.test_strategy",
+    "tests.sentry.stacktraces.test_in_app_normalization",
```

Comment:
this is very easy to fix:

```console
$ mypy tests/sentry/stacktraces/test_in_app_normalization.py 
tests/sentry/stacktraces/test_in_app_normalization.py:90: error: Value of type "Collection[str]" is not indexable  [index]
tests/sentry/stacktraces/test_in_app_normalization.py:128: error: Value of type "Collection[str]" is not indexable  [index]
tests/sentry/stacktraces/test_in_app_normalization.py:242: error: Value of type "Collection[str]" is not indexable  [index]
tests/sentry/stacktraces/test_in_app_normalization.py:244: error: Value of type "Collection[str]" is not indexable  [index]
tests/sentry/stacktraces/test_in_app_normalization.py:246: error: Value of type "Collection[str]" is not indexable  [index]
tests/sentry/stacktraces/test_in_app_normalization.py:248: error: Value of type "Collection[str]" is not indexable  [index]
Found 6 errors in 1 file (checked 1 source file)
```

the easiest fix is to annotate the one big dictionary as `dict[str, Any]`

Reply:
Yeah, in this case I guess you're right, since it's just test data I'm creating on the spot. In all of the real work I've been doing with this code, the data here is typed as `NodeData` and trying to fix that turns out to be a big can of worms, so I think I just wrote off trying to fix types for these functions as something to be done as a separate project. I'll add the annotations.